altair/examples/scatter_alternate_axes_scale.py

```diff
@@ -0,0 +1,36 @@
+"""
+Scatter Plots with alternate Y axis scale
+---------------------------------
```

Comment:
This should be the same length as the line above it to avoid RST errors when building the docs